src/Test/TestCases.Xaml/TestCases.Xaml.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We need to do smth about the XamlTest projects. Perhaps rename the existing one to TestCases.Workflows?

Reply:
Let me handle that in a separate PR. Created issue #76